src/com/dre/brewery/recipe/BRecipe.java

```diff
@@ -36,6 +36,8 @@ public class BRecipe {
 	private String[] name;
 	private boolean saveInData; // If this recipe should be saved in data and loaded again when the server restarts. Applicable to non-config recipes
 	private String optionalID; // ID that might be given by the config
+	private List<String> description; // The text that shows when a brew hasn't been crafted yet in the collection
```

Comment:
i would probably call this something like
```suggestion
	private List<String> undiscoveredLore; // The text that shows when a brew hasn't been crafted yet in the collection
```
and change the config stuff to match

Reply:
Implemented